action/entrypoint.sh

```diff
@@ -34,9 +34,10 @@ if [[ $UNITY_EXIT_CODE -eq 0 ]] || [[ $UNITY_EXIT_CODE -eq 1 ]]; then
   echo "#        Succeeded        #"
   echo "###########################"
   echo ""
-  echo "Use the file \"$FILE_PATH\" for manual activation."
-  echo ""
-  echo "Set the contents of the resulting license file as the \$UNITY_LICENSE variabe."
+  echo "1. Use the file \"$FILE_PATH\" for manual activation at https://license.unity3d.com/manual"
+  echo "2. Create a GitHub secret with the content of the resulting license file"
+  echo "3. set the \$UNITY_LICENSE variable to your newly created secret"
+  echo "For more information see https://game.ci/docs/github/activation"
```

Comment:
Perhaps it's better to leave out all the details beyond saying what the file path is.

The previous description is not 100% correct, because we found that it may be unsafe to set `UNITY_LICENSE` as the workflows env variable. The new description gives more detail on how to do that, but perhaps it's better to directly refer back to the docs.

I would suggest to replace the old line 39 with your new line 40.